dashboard/config/environments/test.rb

```diff
@@ -31,12 +31,6 @@
     # config.assets.js_compressor = :uglifier
     # config.assets.css_compressor = :sass
 
-    # Do not fallback to assets pipeline if a precompiled asset is missed.
-    config.assets.compile = false
-
-    # Generate digests for assets URLs.
-    config.assets.digest = true
-
```

Comment:
these removed lines had no effect because they were "conditionally" setting these to configuration parameters to their default values. This means that we've actually already been using digested assets in drone ui tests.